docs/editors/debug-adapter-protocol.md

```diff
@@ -15,16 +15,16 @@ capabilities exposed by the client.
 ### Via code lenses
 
 The editor needs to handle two commands in its language client extension:
-[`metals-run-session-start`](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala#L56)
+[`metals-run-session-start`](https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala#L56)
 and
-[`metals-debug-session-start`](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala#L78).
+[`metals-debug-session-start`](https://github.com/scalameta/metals/blob/main/metals/src/main/scala/scala/meta/internal/metals/ClientCommands.scala#L78).
 Those commands should get executed automatically by the lsp client once the user
 activates a code lens. The difference between them is that the former ignores
 all breakpoints being set while the latter respects them. The procedure of
 starting the run/debug session is as follows:
 
 Then we can request the debug adapter uri from the metals server using the
-[`debug-adapter-start`](https://github.com/scalameta/metals/blob/master/metals/src/main/scala/scala/meta/internal/metals/ServerCommands.scala#L95)
```

Comment:
this line is no longer valid as content was shifted by adding `CleanCompile` command. I know it is completely unrelated to this PR but just noticed it :)

Reply:
ah, good catch. I'm actually planning on spending a bit of time today reading through the docs since there are a few things I noticed weren't small mistakes. I'll include this change in there.